code/headers/frame_handler.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <array>
+#include <display_ostream.hpp>
+
+#include <battery_frame_action.hpp>
+#include <distance_frame_action.hpp>
+#include <manual_control_frame_action.hpp>
+#include <temperature_frame_action.hpp>
+
+namespace r2d2::robos {
+    class frame_handler_c {
+    private:
+        base_comm_c &comm;
+
+        display_ostream_c power_ostream{
```

Comment:
Are you sure there isn't a better way to handle the different frames than to hardcode them? Doing it this way means that there has to be a change to the class every single time a new frame is added...

Reply:
According to the previous team this could be changed, by using static members. Our team is planning to change the structure of robOS as seen in the presentation. So this should be improved in a future release of robOS.